packages/sdk/src/protocols/filter.ts

```diff
@@ -314,6 +328,20 @@ class FilterSDK extends BaseProtocolSDK implements IFilterSDK {
   ): Promise<IAsyncIterator<T>> {
     return toAsyncIterator(this, decoders);
   }
+
+  private getPubsubTopics(decoders: IDecoder<any> | IDecoder<any>[]): string[] {
```

Comment:
Can we provide the `DecodedMessage` type here for `IDecoder` instead of `any`?

Reply:
Ideally `unknown` should be used as we don't care about internal - but for some reason `ts` doesn't like it

Reply:
fixed